pkg/ipcmd/ipcmd.go

```diff
@@ -2,23 +2,16 @@
 package ipcmd
 
 import (
+	"fmt"
 	"regexp"
 	"strings"
 
 	"github.com/openshift/openshift-sdn/pkg/exec"
 )
 
-var ipcmdPath string
 var addressRegexp *regexp.Regexp
 
 func init() {
-	var err error
-
-	ipcmdPath, err = exec.LookPath("ip")
```

Comment:
We *expect* it, yes, but we can't guarantee it in all cases.

Returning an error here means that openshift won't start up if it's not present, which is good